### DISCUSSION TITLE
Huge windows 11 (rare) stability system bug with 23.2-qt6-Win64. "no toolbar" "windows flickering" & how to solve it

### DISCUSSION THREAD

**Discussion by Angobrig** [Stellarium/stellarium#3385] [General]
Hi !  

The last 23.2-qt6-Win64 can occur a rare but important bug on Windows 11. I can't tell if it's Stellarium who reach the conditions or if it's the current Windows 11 state who messed up.  Sorry for my english, i'm not english native. 

:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: PROBLEM ::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::

I currently reinstall a new pc and i got right after Stellarium a system and graphic bug who put away the toolbar. The whole windows was flickering on the whole screen through icons. 
Nearly nothing was responding and the mouse was constantly buffering. The whole system was like constantly reseting all. 
Off course, Windows shorcut weren't reponding either. 
So the system was nearly completly blocked. 

I did some research on internet and it seem only a bunch of people encounter since the last 3 or 2 year an exact same problem. But probably from different issue. Of course, the only splendid solution was " reinstall your whole computer " .... Hahem ....

::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: HOW MY INSTALLATION LEAD TO THIS ISSUE ::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::

I ended in this trap by : 
Installing the last "23.2-qt6-Win64" on "administrator mode" on the second pannel. 
I choose to install it with only 1 shorcut for my account and not all. 

Yep Hell don't ask too much nowadays.

:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: WHAT WAS REALLY HAPPENING IN SERVICES :::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::

In fact, the task gestionary, showed me that "Windows explorer" was constantly blinking. It was a little bit special. 

1 "Windows explorer" was on the run

2 A second one named "explorer" show up. This services wasn't mark contrary to "Windows explorer" as a "system" services 

3 "Windows explorer" cease to work on the task gestionary.

4  The system services  "Windows explorer" relaunch but as only "Explorer" on his name. Like the old windows version. 

5 Same scenario, this system services ceased to work or is put down by the second process. And it was working like this constantly. 

The average time of theses permutation was around 1 or 2 second delay.  

Of course you can nearly not write above or in the command. It nearly constantly deselect all. 
Same nearly all stuff on services process don't work. 
At some try you can just end the explorer services but it don't help to solve the issue. 

::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: HOW TO SOLVE THE ISSUE :::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::

Keep in mind that nothing else on the computer launch or work, as Windows was constantly relaunch the explorer process ( and failed )

1  So "CTRL" + "ALT" + "SUPPR" 

2  Task gestionary < Process section 

3  Execute a New task   above 

4  browse < Open the hardrive were Stellarium is installed and open the uninstaller < Unistall ALL of Stellarium

Bingo ! No more issue. The big bug is solved as simple as that.  No resinstallation needed. Just restart your computer. 

::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::HOW I REINSTALL STELLARIUM WITHOUT ISSUES ::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::

Allright ! Prepare to the dead . Winter is coming !

1 Run Stellarium 23.2-qt6-Win64 without administrator rights

2 Don't install any icon on any way

                                                 ,::::.._
                                               ,':::::::::.
                                           _,-'`:::,::(o)::`-,.._
                                        _.', ', `:::::::::;'-..__`.
                                   _.-'' ' ,' ,' ,\:::,'::-`'''
                               _.-'' , ' , ,'  ' ,' `:::/
                         _..-'' , ' , ' ,' , ,' ',' '/::
                 _...:::'`-..'_, ' , ,'  , ' ,'' , ,'::|
              _`.:::::,':::::,'::`-:..'_',_'_,'..-'::,'|
      _..-:::'::,':::::::,':::,':,'::,':::,'::::::,':::;
        `':,'::::::,:,':::::::::::::::::':::,'::_:::,'/
        __..:'::,':::::::--''' `-:,':,':::'::-' ,':::/
   _.::::::,:::.-''-`-`..'_,'. ,',  , ' , ,'  ', `','
 ,::SSt:''''`                 \:. . ,' '  ,',' '_,'
                               ``::._,'_'_,',.-'
                                   \\ \\
                                    \\_\\
                                     \\`-`.-'_
                                  .`-.\\__`. ``
                                     ``-.-._
                                         `

Done. BIG BIG issues for nearly nothing

Well if it help that’s all that matters.

**Comment by gzotti** [comment]
Have you rebooted Windows at least once after this incident, and tried launching Stellarium after that reboot? Maybe some system update messed something up, but there are too many possibilities to poke in possibility space without even a logfile.

**Reply by Angobrig** [reply]
Hi ! I reboot my computer several times at first but it never changed this bug nor calm its effect for a little time. 
I'm on W11 professionnal and on the last update given by microsoft not the beta one : 
- 2023-04 KB4023057
-2023-08  KB5029263
-2023-08 KB5028948 
But i don't think it will lead to anything as theses people on internet with the exact same bug begin to claim since 2 years ago at list. 

As i see it, Stellarium on his actual config can only trigger a problem within W11 code.  2 Different explorer process at the same time, it's really weird. 
Well i can't promess anything but i will take a look on theses logfile. At list a basic solution exist now.
